node/narwhal/examples/simple_node.rs

```diff
@@ -398,6 +399,35 @@ async fn start_server(bft: Option<BFT<CurrentNetwork>>, primary: Primary<Current
 
 /**************************************************************************************************/
 
+/// The operating mode of the node.
+#[derive(Debug, Clone, ValueEnum)]
+enum Mode {
+    /// Runs the node with the Narwhal memory pool protocol.
+    Narwhal,
+    /// Runs the node with the Bullshark BFT protocol (on top of Narwhal).
+    Bft,
+}
+
+/// A simple CLI for the node.
+#[derive(Parser, Debug)]
+struct Args {
+    /// The mode to run the node in.
+    #[arg(long)]
+    mode: Mode,
+    /// The ID of the node.
+    #[arg(long, value_name = "ID")]
+    node_id: u16,
+    /// The number of nodes in the network.
+    #[arg(long, value_name = "N")]
+    num_nodes: u16,
+    /// If set, the path to the file containing the committee configuration.
+    #[arg(long, value_name = "PATH")]
+    config: Option<PathBuf>,
+    /// Enables the tx and solution cannons, and optionally the interval in ms to run them on.
+    #[arg(long, value_name = "INTERVAL_MS")]
+    fire_cannons: Option<Option<u64>>,
```

Comment:
nit: I'd prefer this be broken up into `fire-solutions` and `fire-transactions`. We can have a catch all that fires all with `fire-transmissions` or `fire-all` (you're preference between those two)

Reply:
Done, went with `fire-transmissions` to be consistent with our internal naming for the algebraic type.